decentraland/unity-renderer
# Fix: fixed avatar renderer 2D lod rotation

## What does this PR change?

Fix #2013
Sets correctly the rotation of 2D low LOD players

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/avatar-lod
2. Teleport to a crowded area
3. Verify the correct rotation of low LOD players that are far

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
